Doc/distutils/setupscript.rst

```diff
@@ -524,19 +524,22 @@ following way::
     setup(...,
           data_files=[('bitmaps', ['bm/b1.gif', 'bm/b2.gif']),
                       ('config', ['cfg/data.cfg']),
-                      ('/etc/init.d', ['init-script'])]
          )
 
+Each (*directory*, *files*) pair in the sequence specifies the installation
+directory and the files to install there.
 Note that you can specify the directory names where the data files will be
 installed, but you cannot rename the data files themselves.
 
-Each (*directory*, *files*) pair in the sequence specifies the installation
-directory and the files to install there.  If *directory* is a relative path, it
-is interpreted relative to the installation prefix (Python's ``sys.prefix`` for
-pure-Python packages, ``sys.exec_prefix`` for packages that contain extension
-modules).  Each file name in *files* is interpreted relative to the
-:file:`setup.py` script at the top of the package source distribution.  No
-directory information from *files* is used to determine the final location of
+The *directory* should be a relative path.
+It is interpreted relative to the installation prefix
+(Python's ``sys.prefix`` for system installations; ``site.USER_BASE`` for user installations).
+Each file name in *files* is interpreted relative to the
+:file:`setup.py` script at the top of the package source distribution.
+Distutils allows *directory* to be an absolute installation path,
```

Comment:
Move the sentence in the `directory` description above? i.e, have two paragraphs: one describing `directory`, the other describing `files`?

Reply:
I didn't quite follow what you were saying here, but I did organize the text in two paragraphs: the first about `files`, the second about `directory`.